From cf2678a02cedac743ddd00502fc390731a337301 Mon Sep 17 00:00:00 2001
From: Flopster101 <nahuelgomez329@gmail.com>
Date: Thu, 18 Jun 2026 15:35:59 -0300
Subject: [PATCH] s5e8825-common: Match libsec-ril protobuf SONAME

Backport the Android.bp dependency correction from the upstream protobuf 21.7
compatibility change. The pinned FYH2 libsec-ril blob has a DT_NEEDED entry for
libprotobuf-cpp-full-21.7.so, and the pinned vendor tree already provides that
module.

Upstream-Commit: cf2678a02cedac743ddd00502fc390731a337301
Signed-off-by: Flopster101 <nahuelgomez329@gmail.com>
---
 Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.bp b/Android.bp
index 9973d3a..e947080 100644
--- a/Android.bp
+++ b/Android.bp
@@ -1142,7 +1142,7 @@ cc_prebuilt_library_shared {
                 "libjsoncpp",
                 "libVendorSemTelephonyProps",
                 "libVendorSemDataProps",
-                "libprotobuf-cpp-full",
+                "libprotobuf-cpp-full-21.7",
                 "libvkmanager_vendor",
                 "libengmode_client",
                 "libbase",
-- 
2.51.0

